packages/modal/src/Modal.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import IconClose from 'mdi-react/CloseIcon';
```

Comment:
Better if use SVG file for icons (and even better, if you can transfer the icon through the props)

Reply:
Better, if you can transfer the icon through the props.
- Agree

![image](https://user-images.githubusercontent.com/17886232/72246962-94a39280-35fc-11ea-89a6-109a0fb4a094.png)
- You mean by default add something like this instead of import it from mdi/react?
- If yes, then i don't see an advantage.

Reply:
> You mean by default add something like this instead of import it from mdi/react?
Yes. 

The first argument is mdi-react is a very large library for the use of only a single file.
 Also, mdi-react is updated very often (the minor version changes every month, major can twice a year), and I had a problem with different versions of these libraries.